addon/styles/components/ui-pagination--default-pages.scss

```diff
@@ -32,7 +32,3 @@
     }
   }
 }
-
-.ui-pagination--default-pages {
-  @include ui-pagination--default-pages();
-}
```

Comment:
@jacojoubert also don't need to declare these classes anymore :) I'm working on automating the mixin name as well (in followup pull)